cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -243,6 +242,9 @@ func (p *BicepProvider) Deploy(
 				bicepDeploymentData.Target,
 				bicepDeploymentData.Template,
 				bicepDeploymentData.Parameters,
+				map[string]*string{
+					"azd-env-name": to.Ptr(p.env.GetEnvName()),
```

Comment:
Let's do refactor `azd-env-name` as a shared constant if possible. An accidental disagreement between implementations results in very unexpected behavior for the user.

Reply:
Added the following constants to a new `tags.go` in the `azure` package:

	// TagKeyAzdEnvName is the name of the key in the tags map of a resource
	// used to store the azd environment a resource is associated with.
	TagKeyAzdEnvName = "azd-env-name"
	// TagKeyAzdServiceName is the name of the key in the tags map of a resource
	// used to store the azd service a resource is associated with.
	TagKeyAzdServiceName = "azd-service-name"

And updated the uses of these fixed strings in the codebase to use them.

Reply:
Love it!